app/utils/ui/ui.ts

```diff
@@ -96,4 +97,8 @@ const getIconFromModelType = (emissionModelType) => {
   }
 };
 
-export default { getTranslationModelType, getIconFromModelType };
+const getColorScheme = () => {
```

Comment:
Instead of `getColorScheme`, could you export `isDarkModeEnabled` so don't have to do the colorScheme==="dark" everytime?

Reply:
Okay